src/hooks/useOrientation/index.ts

```diff
@@ -8,14 +8,15 @@ interface UseOrientationProps {
 
 const useOrientation = ({onOrientationChange}: UseOrientationProps = {}) => {
   const [orientation, setOrientation] = useState(Constants.orientation);
+  const listener = useRef<any>();
 
   const orientationChangeListener = useCallback(() => {
     setOrientation(Constants.orientation);
   }, []);
 
   useEffect(() => {
-    Constants.addDimensionsEventListener(orientationChangeListener);
-    return () => Constants.removeDimensionsEventListener(orientationChangeListener);
+    listener.current = Constants.addDimensionsEventListener(orientationChangeListener);
```

Comment:
I don't think we need the `useRef` here:
```
const listener = Constants.addDimensionsEventListener(orientationChangeListener);
```

Reply:
Good point 👌 
Fixed